metadata.json

```
@@ -22,7 +22,7 @@
     },
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.13.0 < 9.0.0"
+      "version_requirement": ">= 4.13.0 < 10.0.0"
```

The namespaced version of merge() was added in [04d1f0c5cccb93d0bc928c7623c4b0c0ebda5b6b](https://github.com/puppetlabs/puppetlabs-stdlib/commit/04d1f0c5cccb93d0bc928c7623c4b0c0ebda5b6b) which require version 9.0.0 of the stdlib module:

```suggestion
      "version_requirement": ">= 9.0.0 < 10.0.0"
```

---

updated accordingly, thanks